### PR TITLE
Add create project flow to project setup guide

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description
This PR adds steps to create a project to the project setup guide

# Reasons
GitLab and GitHub apps users need to create a standalone project and connect that to a code repo, which is a change from the existing method of setting up a project that mirrors a repo in an org

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/create-project.adoc [hunk 1]
```diff
@@ -12,40 +12,41 @@ contentTags:
...
 
+* A CircleCI account connected to your code. You can link:https://circleci.com/signup/[sign up for free].
+* A code repository you want to build on CircleCI.
 
-[#step-one-link-your-vcs-with-circleci]
-== Step One: Link your VCS with CircleCI
+== Build a new project on CircleCI
 
-If you have not done so already, <<first-steps#,sign up with CircleCI>> and select your Version Control System (VCS). You can also sign up with email.
+The authentication method used to set up your CircleCI account determins the definition of "project" in CircleCI, as well as the permissions management processes available to you:
```
Comment (kbatuigas) [inline jekyll/_cci2/create-project.adoc:25]:
```suggestion
The authentication method used to set up your CircleCI account determines the definition of "project" in CircleCI, as well as the permissions management processes available to you:
```

File: jekyll/_cci2/create-project.adoc [hunk 2]
```diff
@@ -12,40 +12,41 @@ contentTags:
...
+* If you authenticated using **GitHub OAuth**, or **Bitbucket**, a "project" in CircleCI maps directly to a "repository" in GitHub. Follow steps to <<set-up-a-project>> below.
+* If you authenticated through **GitLab**, a "project" within CircleCI is a standalone entity that you create and then associate with a code repository. Follow steps to <<create-a-project>> below.
 
-[#step-two-create-a-project-in-circleci]
-== Step Two: Create a project in CircleCI
+[#set-up-a-project]
+=== Set up a project
 
-Follow these steps to create a new project in CircleCI:
+NOTE: If you authenitcated CircleCI with either xref:github-integration#[GitHub OAuth], or xref:bitbucket-integration#[Bitbucket], the steps in this section apply to you.
```
Comment (kbatuigas) [inline jekyll/_cci2/create-project.adoc:33]:
```suggestion
NOTE: If you authentecated CircleCI with either xref:github-integration#[GitHub OAuth], or xref:bitbucket-integration#[Bitbucket], the steps in this section apply to you.
```

File: jekyll/_cci2/create-project.adoc [hunk 3]
```diff
@@ -58,16 +59,48 @@ For guidance on creating a `config.yml` file, see <<config-intro#,Configuration
 +
 . Click the blue **Set Up Project** button.
 
-CircleCI then uses the specified `config.yml` file to run your pipeline. You can see the output in the CircleCI dashboard.
+CircleCI uses the specified `.circleci/config.yml` file to run your pipeline. You can see the output in the CircleCI dashboard.
 
-To make changes to your pipeline, either edit the `.circleci/config.yml` file in your repo, or click the ellipsis next your project in the Dashboard and choose **Configuration File**.
+To make changes to your pipeline, either edit the `.circleci/config.yml` file in your repo, or click the ellipsis next your project in the Dashboard and choose **Configuration File**. This opens the CircleCI Configuration Editor, from where you can edit and commit your `config.yml` file.
```
Comment (kbatuigas) [inline jekyll/_cci2/create-project.adoc:64]:
```suggestion
To make changes to your pipeline, either edit the `.circleci/config.yml` file in your repo, or click the ellipsis next your project in the Dashboard and choose **Configuration File**. This opens the CircleCI configuration editor, from where you can edit and commit your `config.yml` file.
```

File: jekyll/_cci2/create-project.adoc [hunk 4]
```diff
@@ -58,16 +59,48 @@ For guidance on creating a `config.yml` file, see <<config-intro#,Configuration
...
 
 image::edit-config-file.png[Edit Configuration File]
 
-This opens the CircleCI Configuration Editor, from where you can edit and commit your `config.yml` file.
+[#create-a-project]
+=== Create a project
+
+NOTE: If you authenitcated CircleCI with either xref:gitlab-integration#[GitLab], or xref:github-apps-integration#[GitHub apps], the steps in this section apply to you.
```
Comment (kbatuigas) [inline jekyll/_cci2/create-project.adoc:71]:
```suggestion
NOTE: If you authenticated CircleCI with either xref:gitlab-integration#[GitLab], or xref:github-apps-integration#[GitHub apps], the steps in this section apply to you.
```

File: jekyll/_cci2/create-project.adoc [hunk 5]
```diff
@@ -12,40 +12,41 @@ contentTags:
...
-[#step-one-link-your-vcs-with-circleci]
-== Step One: Link your VCS with CircleCI
+== Build a new project on CircleCI
 
-If you have not done so already, <<first-steps#,sign up with CircleCI>> and select your Version Control System (VCS). You can also sign up with email.
+The authentication method used to set up your CircleCI account determins the definition of "project" in CircleCI, as well as the permissions management processes available to you:
 
-A CircleCI project must be linked with an existing repo in your VCS. Make sure you have a repo and that you have authorized CircleCI to access it (in GitHub, you have the option to block CircleCI from accessing your private repositories).
+* If you authenticated using **GitHub OAuth**, or **Bitbucket**, a "project" in CircleCI maps directly to a "repository" in GitHub. Follow steps to <<set-up-a-project>> below.
+* If you authenticated through **GitLab**, a "project" within CircleCI is a standalone entity that you create and then associate with a code repository. Follow steps to <<create-a-project>> below.
```
Comment (kbatuigas) [inline jekyll/_cci2/create-project.adoc:28]: Do we need to also mention GitHub apps here? We do it in the "Create a project" section

**Review by kbatuigas — CHANGES_REQUESTED** [review]
Just a couple of typos; otherwise looks great!

File: jekyll/_cci2/create-project.adoc (hunk 5 above)
Comment (rosieyohannan) [reply jekyll/_cci2/create-project.adoc]: Done, I had forgotten to push this up but sone now

**Review by kbatuigas — APPROVED** [review]
🎉

**Pull request merged**